File: Week2/SourceCode/ManageStudent.java

import java.util.Scanner;

public class ManageStudent {
    public static void main(String fdxzfdxf[]){
        StudentProfile std1 = new StudentProfile(6088130, "Sunat", 18, "123456asdsad");
        StudentProfile std2 = new StudentProfile(6088124, "Chawna", 2, "chsadsad55474");
        std1.getAllinfo();
        std2.getAllinfo();
        agediff(std1,std2);
    }
    public static void agediff (StudentProfile std1, StudentProfile std2)
    {
    	System.out.println(Math.abs(std1.getAge()-std2.getAge()));
    	
    }
    
}

File: Week 2/bin/SourceCode/ManageStudent.kt
import java.util.Scanner

object ManageStudent {
    @JvmStatic
    fun main(fdxzfdxf: Array<String>) {
        val std1 = StudentProfile(6088130, "Sunat", 18, "Bankrub")
        val std2 = StudentProfile(6088124, "Chawna", 2, "Bangrachun")
        std1.getAllinfo()
        std2.getAllinfo()
        agediff(std1, std2)
    }

    fun agediff(std1: StudentProfile, std2: StudentProfile) {
        print("age diff is ") == println(Math.abs(std1.age - std2.age))

    }

}